MAUI/Autocomplete/Getting-Started.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Autocomplete | Syncfusion
+description: Learn how to get started with Syncfusion .NET MAUI Autocomplete control and its suggestion list, as well as other topics, in this section.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Autocomplete (SfAutocomplete)
+
+This section explains how to add the Autocomplete control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion `Autocomplete` control.
+
+## Adding a .NET MAUI Autocomplete reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add .NET MAUI Autocomplete to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.Inputs` and then install it.
+
+## Handler registration 
+
+ In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# hl_lines="6 7 18 19" %}
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+using Syncfusion.Maui.ListView.Hosting;
+
+namespace AutocompleteSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureSyncfusionListView()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}     
+
+## Create a Simple .NET MAUI Autocomplete
+
+The .NET MAUI Autocomplete control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a .NET MAUI Autocomplete (SfAutocomplete) and configure its elements.
+
+### Adding the .NET MAUI Autocomplete control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample.
```

Comment:
Step 2: Add the namespace as shown in the following code sample:

Reply:
Changes are made.